source/_integrations/zha.markdown

```diff
@@ -147,12 +147,12 @@ Press `Submit`. The success dialog will appear or an error will be displayed in
 
 ### ZiGate or Sonoff ZBBridge Devices
 
-If you are use ZiGate or Sonoff ZBBridge you have to use some special usb_path configuration:
+If you are using ZiGate Ethernet/WiFi Gateway, Sonoff ZBBridge, or other network-attached Zigbee adapter you will have to use some special usb_path configuration. It is also strongly advised that you use a reserved or static IP Address is strongly by reserving the IP address in your router and/or configure a static IP address in your Zigbee gateway/bridge.
```

Comment:
What makes ZHA different compared to any other device/solution that requires this to be highlighted?
IMHO, this is not specific to this integration, additionally, most people won't have a clue what you are talking about in this regard.

Reply:
It is different because Zigbee bridges for ZHA like these  are really only used as network-attached remote serial port proxy adapters, like ser2net, and streaming serial communication like this is extremely sensitive to communication loss. 

So you can not compare this connection to something like MQTT or other protocols that are designed to handle communication loss.

Reply:
I don't see how I cannot compare this. Additionally, if a connection fails to a remote, the integration should help to user to update the connection settings by triggering the flow.

That said, I still don't get how this is different from e.g., a Hue hub, or a remote MQTT server or my deCONZ machine, or remote Z-WaveJS2MQTT that is integrated without MQTT but over Z-Wave JS directly...

Reply:
> It is very different from all those you mentioned there because with the ZHA integration the hub/gateway software that communicates directly via the microcontroller-unit's serial/UART port is not translated to the other transport protocol.

The protocol itself is not part of the IP communication. Those are the same for all sketched. My above answer stays the same.

Reply:
Zigbee MCU <-> Serial/UART connection <-> ZHA integration

vs. 

Zigbee MCU <-> Serial/UART connection<-> Zigbee2MQTT <-> MQTT Broker <-> MQTT Client

Zigbee MCU <-> Serial/UART connection <-> Philips Hue Hub <-> Philips Hue API <-> Philips Hue API integration

Zigbee MCU <-> Serial/UART connection <-> deCONZ <-> deCONZ API <-> deCONZ API integration

Reply:
See my above comments.

Reply:
It is still not correct. Connections via serial stream proxies are direct serial connections as servers like ser2net is only a tunnel.

Please consider getting a second opinion.

Reply:
I don't need to consider a second opinion. THe integration needs to handle IP changes correct and guide the user in the UI in those cases. This adds a lot of details that might confuse the user and it no different from other situations (e.g., DSMR, and all others I mentioned).

If ZHA doesn't handle the change correct or doesn't guide the user correct on the IP change of the gateway/bridge connecting too, that should be filed as a bug. I'm not willing to accept this addition.

Reply:
> THe integration needs to handle IP changes correct and guide the user in the UI in those cases.

Well the ZHA integration does not handle IP changes for Zigbee coordinators remotely connected via socat today. So bug?

Reply:
Well, yes, that should be handled correctly. Network changes happen and should be taken into account.

Reply:
Zha integration handles up changes for tubezb coordinator. If user went all the way to setup zha coordinator to use ipserial tunnel, he may as well be smart enough to reserve an IP address without hand holding.

Reply:
I agree with @frenck .
The handling of a IP change is not unique to this integration.